microsoft-edge/webview2/resources.md

```diff
@@ -0,0 +1,42 @@
+---
+description: Resources, talks, videos, and blogs about Microsoft Edge WebView2
+title: Resources, talks, videos, and blogs about WebView2
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 08/09/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
+---
+# Resources, talks, videos, and blogs about WebView2
+
+These external resources cover Microsoft Edge WebView2.
+
+
+## Videos of talks
+
+
+| Title | Description | Length and date |
+|---|---|---|
+| [Microsoft Edge \| Building, Deploying, and Managing WebView2 Applications](https://www.youtube.com/watch?v=LX-eXvcSx6c) | From the Microsoft Ignite 2020 conference. | YouTube channel: Microsoft Edge.  Length: 11:04.  Uploaded March 2, 2021. |
+| [Bringing the Best of the Web to Your .NET Windows Apps with WebView2](https://www.youtube.com/watch?v=I_ZBhK9_gTE) | From .NET Conf 2021 - Focus on Windows. | YouTube channel: dotNET.  Length: 28:16.  Uploaded March 1, 2021. |
+| [WebView2: Bringing the best of the web to your native apps](https://www.youtube.com/watch?v=-ri7TmPeqLc) | From the Microsoft Ignite 2020 conference. | YouTube channel: Microsoft Edge.  Length: 13:36.  Uploaded September 22, 2020. |
+| [Announcing Microsoft Edge WebView2 for Office Add-Ins Platform](https://www.youtube.com/watch?v=D73D-EB89Fs) | How we are evolving the web viewer control used by the Office add-ins platform for add-ins running on Windows.  A talk from Office Add-Ins. | YouTube channel: Microsoft 365 Developer.  Length: 5:07.  Uploaded August 12, 2020. |
+| [WebView2: Bringing the best of the web to your native apps \| BDL105](https://www.youtube.com/watch?v=P0K3DWp05QQ) | From Microsoft Build 2020. | YouTube channel: Microsoft Developer.  Length: 14:37.  Uploaded May 24, 2020. |
+  
+About the Ignite conference at which some of these videos were presented: [Microsoft Ignite 2020](https://news.microsoft.com/ignite2020/): Microsoft's annual gathering of technology leaders and practitioners.
```

Comment:
Only one comment on this page ... two colons in one sentence. I don't know if there's any way around this, but if feels awkward.

Reply:
Way around this: delete and streamline to concentrate attention on the video title.